e2e/tests/0060-snapshots.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
```

Comment:
I know this test is already written, but we plan on removing `e2e/` test in favor of robot framework tests in `test/` so it'd be nice to have it there

Reply:
Sounds good. The test is pretty trivial to implement.

Reply:
No, better to just implement the RF test now and be done with it.